dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportControllerTest.java

```diff
@@ -262,6 +263,30 @@ void shouldGetEnrollmentWithRelationshipsWhenRelationshipIsDeletedAndIncludeDele
         () -> assertHasMember(jsonRelationship, "bidirectional"));
   }
 
+  @Test
+  void
+      shouldGetEnrollmentWithNoRelationshipsWhenEnrollmentIsOnToEndOfAUnidirectionalRelationship() {
```

Comment:
I'm not sure I'm reading this right, did you mean to say `WhenEnrollmentIsOn**The**End`?

Reply:
That was confusing.
I renamed it to `shouldGetEnrollmentWithNoRelationshipsWhenEnrollmentIsOnTheToSideOfAUnidirectionalRelationship`